components/compliance-service/inspec/cli.go

```diff
@@ -299,6 +304,10 @@ func getInspecError(stdOut string, stdErr string, err error, target *TargetConfi
 		}
 	}
 
+	if err == nil {
```

Comment:
don't we already cover this on line 321 with the return?

Reply:
if err is nil, line 321 will not be reached as the service will crash on line 311.

Reply:
ahh ya - 👍